### ISSUE TITLE
Problem variants?

### ISSUE THREAD

**Issue by smlmbrt** [PGScatalog/pgs-harmonizer#23]
Some reports:
> variants with a valid rsid tags but an "Unknown" source reference tag or “NR” genome_build tag, though it also occurs in some novel variants. For example PGS003753_hmPOS_GRCh38.txt.gz rs147937400 (T>C), which seems to correspond to an original reference of GRCh37 (chr6 29068745), but appears unmapped in the PGS Catalog’s GRCh38 harmonized score file (UCSC reports it mapped to chr6 29100968 in GRCh38)

Related to https://github.com/PGScatalog/pgsc_calc/discussions/137

**Comment by smlmbrt** [comment]
@ens-lgil can we look at which it seems to miss that rsID? It seems 104/35445 don't map and get "Unknown" but at least rs147937400 should be mapped to chr6 via ensembl: https://www.ensembl.org/Homo_sapiens/Variation/Explore?db=core;r=6:29100468-29101468;v=rs147937400;vdb=variation;vf=193308664

**Comment by ens-lgil** [comment]
Yes, it seems that it's an issue with the UCSC liftover. We need to find an other mapping source when this happens.

**Comment by smlmbrt** [comment]
But that one should have been mapped by the rsID to chr6, not the liftover.

**Comment by ens-lgil** [comment]
This PR should fix at least partially the issue: https://github.com/PGScatalog/pgs_variants_coords/pull/4

**Comment by smlmbrt** [comment]
Will it require re-running everything?

**Comment by ens-lgil** [comment]
Yes, just to be on the safe side
